packages/lego-bricks/src/components/Accordion/index.tsx

```diff
@@ -26,26 +27,35 @@ export const Accordion = ({
   disabled = false,
   defaultOpen = false,
   animated = true,
+  removeChildrenOnClose = false,
```

Comment:
Any reason not to leave this as default true? Dunno if there are any places where it is particularly advantageous to keep it rendered when hidden?

Reply:
I was a bit split on this. I tested it mostly with the readme container on the frontpage which contain some images. Whit `removeChildrenOnClose` and animation the opening animation was a bit wonky as the height of children changes.

Still I dont think its that bad, but I havent tested it much other places. I will look into it tomorrow probably.

If it seems to be generally fine I will change it to default to true and maybe just disable it for the readme container in particular.